### PR TITLE
fix: migrate from yarn to pnpm

### PR DESCRIPTION
**Pull request by dgolovin**

### CODE REVIEW

File: .github/workflows/e2e-main.yaml [hunk 1]
```diff
@@ -97,33 +104,20 @@ jobs:
...
+          pnpm install
+          pnpm test:e2e:build
 
-      - name: Execute yarn in Red Hat Account Extension
+      - name: Execute pnpm in Red Hat Account Extension
         working-directory: ${{ env.REPOSITORY }}
         run: |
-          yarn add -D @podman-desktop/tests-playwright@next
-          yarn --frozen-lockfile
+          pnpm add -D @podman-desktop/tests-playwright@1.13.0-202408291153-ccbdd9b
```
Comment (odockal) [inline .github/workflows/e2e-main.yaml:113]: Please revert this line, we need to have @next as it points to the latest released npm package version.

File: .github/workflows/e2e-main.yaml [hunk 2]
```diff
@@ -97,33 +104,20 @@ jobs:
...
+          pnpm build
```
Comment (odockal) [inline .github/workflows/e2e-main.yaml:114]: use `install` as we need to pull deps.

**Review by odockal — CHANGES_REQUESTED** [review]

File: .github/workflows/e2e-main.yaml (hunk 1 above)
Comment (dgolovin) [reply .github/workflows/e2e-main.yaml:113]: Can we keep it for now? I'll update it as part of migrating to use PW runner.

Comment (dgolovin) [inline .github/workflows/e2e-main.yaml:113]:
```suggestion
          pnpm install -D @podman-desktop/tests-playwright@1.13.0-202408291153-ccbdd9b
```

File: .github/workflows/e2e-main.yaml [hunk 3]
```diff
@@ -97,33 +104,20 @@ jobs:
...
+          pnpm install
+          pnpm test:e2e:build
 
-      - name: Execute yarn in Red Hat Account Extension
+      - name: Execute pnpm in Red Hat Account Extension
         working-directory: ${{ env.REPOSITORY }}
         run: |
-          yarn add -D @podman-desktop/tests-playwright@next
-          yarn --frozen-lockfile
+          pnpm install -D @podman-desktop/tests-playwright@1.13.0-202408291153-ccbdd9b
```
Comment (odockal) [inline .github/workflows/e2e-main.yaml:113]: I do no think we can have specific version checksum here. we need `next` as it automatically adds latest version (nightly) in the package.json and installs it afterwards.

File: .github/workflows/pr-check.yaml [hunk 4]
```diff
@@ -133,34 +120,21 @@ jobs:
...
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-yarn-
+          pnpm install
+          pnpm test:e2e:build
 
       - name: Ensure getting current HEAD version of the test framework
         working-directory: ./podman-desktop-redhat-account-ext
-        run: yarn add -D @podman-desktop/tests-playwright@next 
+        run: pnpm add -D @podman-desktop/tests-playwright@1.13.0-202408291153-ccbdd9b
```
Comment (odockal) [inline .github/workflows/pr-check.yaml:128]: same here, please use `next`.

File: .github/workflows/e2e-main.yaml [hunk 5]
```diff
@@ -97,33 +104,20 @@ jobs:
...
+          pnpm build
```
Comment (odockal) [inline .github/workflows/e2e-main.yaml:114]: just installing deps, please use `pnpm install`

**Review by odockal — CHANGES_REQUESTED** [review]
please see the comments.

File: .github/workflows/e2e-main.yaml (hunk 1 above)
Comment (odockal) [reply .github/workflows/e2e-main.yaml]: Ah, I missed your comment. ok. god mode then! :)

**Review by odockal — APPROVED** [review]
We will fix small parts after merging, If I understand it right.

**Pull request merged**